migrations/20161026145654-add-days-of-week.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+module.exports = {
+  up: function (queryInterface, Sequelize) {
+    /*
+      Add altering commands here.
+      Return a promise to correctly handle asynchronicity.
+
+      Example:
+      return queryInterface.createTable('users', { id: Sequelize.INTEGER });
+    */
+    return queryInterface.addColumn(
+      'Channels',
+      'monday',
+      {
+        type: Sequelize.BOOLEAN,
+        defaultValue: true
+      }
+    ).then(() => queryInterface.addColumn(
+      'Channels',
+      'tuesday',
+      {
+        type: Sequelize.BOOLEAN,
+        defaultValue: true
+      }
+    )).then(() => queryInterface.addColumn(
+      'Channels',
+      'wednesday',
+      {
+        type: Sequelize.BOOLEAN,
+        defaultValue: true
+      }
+    )).then(() => queryInterface.addColumn(
+      'Channels',
+      'thursday',
+      {
+        type: Sequelize.BOOLEAN,
+        defaultValue: true
+      }
+    )).then(() => queryInterface.addColumn(
+      'Channels',
+      'friday',
+      {
+        type: Sequelize.BOOLEAN,
+        defaultValue: true
+      }
+    ));
```

Comment:
Not a big deal, but I might do this as a [bit mask](https://en.wikipedia.org/wiki/Mask_(computing)) so you don't have multiple columns for each day. The single column would be an INTEGER or BYTE type. You only need 7 bits to represent all days of the week.